examples/with-prisma-data-provider/prisma/migrations/20230704094414_init/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateTable
```

Comment:
Do the example start with an empty database if you run them, and the user has to create the records themselves?
If that's the case could maybe we could seed some dummy data https://www.prisma.io/docs/guides/migrate/seed-database?